src/coreModules/documentation/components/Nav.js

```diff
@@ -0,0 +1,102 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { compose } from 'redux'
+import { connect } from 'react-redux'
+
+import i18nSelectors from 'coreModules/i18n/globalSelectors'
+import specification from 'dina-schema/build/openApi.json'
```

Comment:
import or require, if it doesn't matter, we use import?

![screenshot 2018-01-09 11 03 53](https://user-images.githubusercontent.com/12738484/34715401-d622d59e-f52c-11e7-9071-07c1da2c2a45.png)

Reply:
Yes. we only use require when we need to be able to run it on the serverside or later want to be able to do that